### ISSUE TITLE
Become a mentor can't increase rating bug

### ISSUE THREAD

**Issue by SsonSAMA** [ValveSoftware/Dota2-Gameplay#12603]
### Description

I just recently noticed that my friend and I couldn't get ratings after becoming instructors for other new players. The score keeps getting stuck at 2555
This is the kind of positive rating I should have gotten as a newbie mentor, but this score hasn't changed since who knows when
This in itself is purely based on our passion to help new players for free, but now that we don't even have reviews anymore, I'd like to resolve this bug

### Screenshots

![2555](https://github.com/ValveSoftware/Dota2-Gameplay/assets/147237094/c9e0f406-e01a-4e0c-b548-3861abf8b4c5)

**Comment by commentator24** [comment]
it's been like this since around march update #8293 so maybe 7 months already

**Comment by SsonSAMA** [comment]
> it's been like this since around march update #8293 so maybe 7 months already please upvote my thread thank you :)

Yes, poor me for working so long for nothing, but the thought of helping new players is just fine, let's hope it gets fixed
